openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -72,9 +72,12 @@ function twitterOn() {
 function isScrolledIntoView(elem) {
     var docViewTop = $(window).scrollTop();
     var docViewBottom = docViewTop + $(window).height();
-    var elemTop = $(elem).offset().top;
-    var elemBottom = elemTop + $(elem).height();
-    return ((docViewTop < elemTop) && (docViewBottom > elemBottom));
+    if ($(elem).offset()) {
```

Comment:
this shouldn't really be in this code review, it should be on the refactor/slick-carousels branch (which updated OL to be ready for slick carousels). The upgrade of jquery makes $(elem) undefined, hence this patch. We might as well add it here. Note: Other things like manage bookcovers breaks w/ latest jquery.